apps/plea/models.py

```diff
@@ -788,6 +788,7 @@ class AuditEvent(models.Model):
         ("case_invalid_duplicate_urn_used", "Invalid Case: Duplicate URN already used (sent)"),
         ("case_invalid_no_offences", "Invalid Case: No offences"),
         ("result_invalid_duplicate_urn_used", "Invalid Result: Duplicate URN already used (sent)"),
+        ("case_invalid_invalid_date", "Invalid Case: Invalid date field")
```

Comment:
Did we find out last time that this counts as needing a migration? (even though it really doesn't)